yt/visualization/volume_rendering/scene.py

```diff
@@ -359,12 +365,12 @@ def save(self, fname=None, sigma_clip=None, render=True):
 
     def save_annotated(
         self,
-        fname=None,
-        label_fmt=None,
-        text_annotate=None,
-        dpi=100,
-        sigma_clip=None,
-        render=True,
+        fname: Optional[str] = None,
+        label_fmt: Optional[str] = None,
+        text_annotate: Optional[str] = None,
+        dpi: int = 100,
+        sigma_clip: Optional[float] = None,
```

Comment:
I just realized if we're going to wait on type hinting until we come up with a community plan maybe we should remove the type`Optional` type hinting here?

Reply:
I'd rather keep this PR waiting, it's not fixing anything important

Reply:
Actually those hints are perfectly compliant with python 3.6 so I don't think we need to keep the PR waiting (of course I still need to handle the failures)
Btw there are _already_ one or two similar type hints in the main branch.

Reply:
My point is that the discussion in the other PR is pausing our type hinting implementation until we have a community discussion about it. This seems to be moving forward with it. 

> Btw there are already one or two similar type hints in the main branch.

I don't see any "from typing" or "import typing" places anywhere in main?

Reply:
Right, but I'm pretty sure one or two type hints were sprinkled in #2981 (@cphyc ?)
Note that some hints are simple enough that they don't require any imports, for instance
```python
def spam(bacon: int, eggs: str) -> float:
    return 1.0
```
that being said I'm having a hard time gripping the ones I think got in already.

Reply:
Not sure if that community discussion happened (I can't find it), but there are several places in the codebase that do `import typing`. Since AFAICT that was the only issue raised with this PR I'm gonna merge it.